content/applications/sales/sales/send_quotations/optional_products.rst

```diff
@@ -1,65 +1,67 @@
-=====================
-Add optional products
-=====================
+=================
+Optional products
+=================
 
-The use of optional products is a marketing strategy for cross-selling products along with a core
-product. The aim is to offer useful and related products to your customers. For instance, if a
-customer wants to buy a car, he has the choice to order an automatic opening trunk and massaging
-seats, or not to order such high-quality products and simply buy his car.
+The use of optional products is a marketing strategy that involves the cross-selling of products
+along with a core product. The aim is to offer useful and related products to customers, which may
+result in an increased sale.
 
-Add optional products to your quotations
-========================================
+For instance, if a customer wants to buy a car, they have the choice to order massaging seats, as
+well, or ignore the offer and simply buy the car. Presenting the choice to purchase optional
+products enhances the customer experience.
+
+Optional products on quotations
+===============================
 
-With Odoo Sales, it is possible to add or modify optional products directly on quotations
-(under the **Optional Products** tab, as you can see below).
+With the Odoo *Sales* application, it is possible to add or modify optional products directly on
+quotations (located under the :guilabel:`Optional Products` tab on a quotation form, in the *Sales*
+application).
 
-.. image:: optional_products/optional_products_2.png
+.. image:: optional_products/optional-products-2.png
```

Comment:
Please update all of the image file names to follow the [image file naming convention](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#media-files). Image names should not be sequential like this because it does not provide any contextual information of the image just by reading the file name